app/src/main/java/com/example/githubusers/model/GithubUsers.java

```diff
@@ -0,0 +1,71 @@
+package com.example.githubusers.model;
+
+import com.google.gson.annotations.SerializedName;
+
+
+public class GithubUsers {
+
+    @SerializedName("username")
+    private String userName;
+
+    @SerializedName("first_name")
+    private String firstName;
+
+    @SerializedName("last_name")
+    private String lastName;
+
+    @SerializedName("public_repos")
+    private int publicRepos;
+
+    @SerializedName("bio")
+    private String bio;
+
```

Comment:
I believe this model class is to hold data for a single GitHub user in the `Github Response` for users. Therefore, be mindful for the available fields in the API Response and also what your Design `Mockup` requires of which I believe is just `username` and `imageUrl` for each user.

Reply:
Sure I will refer to `https://api.github.com/search/users?q=+language:java+location:nairobi&per_page=100` for list view and `https://api.github.com/users/{username}` for detail view